src/writing-your-own-application/index.md

```diff
@@ -4,7 +4,7 @@ With the appropriate Rust compiler and toolchain installed, you're now ready to
 
 You can write an application in the following ways:
 
-- (**Strongly recommended**) Generate from a template: Gives you a configured project, saves time and prevents possible errors.
+- (**Strongly recommended**) Generate from a template: Gives you a configured project, saves time and, prevents possible errors.
```

Comment:
This is a serial comma again. It should come before "and".

Not sure why Grammarly gives this kind of suggestions...